recipes/urduhack/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "urduhack" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f4020332348535c610266f44aa013c70e11b8ecce397ead71f1b847b3a99b4d4
+
+build:
+  number: 0
+  skip: true  # [py >=36]
```

Comment:
This just completely skips the build. We don't build for Python 2 anymore.

Reply:
@xhochy can you please elaborate a bit. You mean I should/must not put   skip: true  # [py >=36] in meta as it will ignore everything and pass the build ?

Reply:
Yes, we only build for Python 3.6+ and this line skips the build for Python >=3.6. I guess this package supports Python 3.6+ and you wanted to skip Python 2? Then you can just omit this.

Reply:
My package is urduhack

Reply:
> Yes, we only build for Python 3.6+ and this line skips the build for Python >=3.6. I guess this package supports Python 3.6+ and you wanted to skip Python 2? Then you can just omit this.

Got it but when I try to build with python3.6 or greater it fails all the test.

Reply:
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
Exception information:
Traceback (most recent call last):
    return _func_defaulting_env_to_os_environ('call', *popenargs, **kwargs)
  File "/usr/local/miniconda/lib/python3.7/site-packages/conda_build/utils.py", line 384, in _func_defaulting_env_to_os_environ
    raise subprocess.CalledProcessError(proc.returncode, _args)
subprocess.CalledProcessError: Command '['/bin/bash', '-o', 'errexit', '/usr/local/miniconda/conda-bld/urduhack_1590668026710/work/conda_build.sh']' returned non-zero exit status